tools/_venv_common.py

```diff
@@ -191,9 +199,27 @@ def install_packages(venv_name, pip_args):
     # Using the python executable from venv, we ensure to execute following commands in this venv.
     py_venv = get_venv_python_path(venv_name)
     subprocess_with_print([py_venv, os.path.abspath('letsencrypt-auto-source/pieces/pipstrap.py')])
+    env_pip_no_binary = os.environ.get('CERTBOT_PIP_NO_BINARY')
+    if env_pip_no_binary:
+        # Check OpenSSL version. If it's too low, don't apply the env variable.
```

Comment:
I don't believe in merging dead code, so I'm down to remove it if it won't be needed. But wasn't the point of doing this because some of our packagers have systems that will have that version of cryptography? I guess since it's not in our own tests this code is useless regardless and I'll remove it, but we should make sure to get that version of cryptography in our tests ASAP so we're covering all environments.

Reply:
This code (or code like it) won't be needed unless upgrade the pinned version of cryptography again before we drop support for Debian 8 and RHEL 6. I think that probably won't happen, but if it does, we can always add this code again.

The point of doing this is to let cryptography be compiled with a new enough version of OpenSSL to hit the problem that hit some of our packagers. I think setting the no binary option does that and doing so should work on every system in the test farm tests now that #8035 has landed.